### PR TITLE
Show @BethanyG on website maintainer list

### PR DESCRIPTION
**Pull request by cmccandless**

### CODE REVIEW

**Review by ErikSchierboom — APPROVED** [review]

**Comment by iHiD** [comment]
(I'd like to verify that @BethanyG actually wants this before it gets merged as many maintainers don't)

**Comment by cmccandless** [comment]
> (I'd like to verify that @BethanyG actually wants this before it gets merged as many maintainers don't)

_Certainly_; I had explicitly requested her review on this for that exact purpose.

**Review by BethanyG — APPROVED** [review]
Thanks for submitting this, @cmccandless.  Yes - I'm happy to have my name listed on the site.  😄

**Pull request merged**
